Sources/SpeziConsent/Resources/ConsentTest.md

```diff
@@ -0,0 +1,32 @@
+---
+study: MHC
+title: MyHeart Counts
+---
+
+# Welcome to MyHeart Counts
+
+Heyyy
+
+- spezi
+- dot
+- stanford
+- dot
+- edu
+
+how about some numbers?
+
+1. one
+2. two
+3. three
+
+<toggle id=toggle1 default=false>Are you sure you have understood the instructions?</toggle>
```

Comment:
What do we do with a Toggle that has no default value? Do we render it as a menu with a "Yes"/"No" selection? Is probably a very common use case that we just want to have a "Yes"/"No".